db/migrate/20231012124745_add_item_type_to_meeting_agenda_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddItemTypeToMeetingAgendaItems < ActiveRecord::Migration[7.0]
+  def change
+    add_column :meeting_agenda_items, :item_type, :integer, limit: 1, default: 0
```

Comment:
We'll want to migrate any existing `MeetingAgendaItem` where work_package is present to the work package type

Reply:
Added.